src/dom-to-image-more.js

```diff
@@ -74,22 +75,46 @@
         const ownerWindow = domtoimage.impl.util.getWindow(node);
         options = options || {};
         copyOptions(options);
+        let restorations = [];
```

Comment:
This is the stack of text nodes we have to restore... there are likely to be ZERO of them, and most likely no more than one, but it's just easier to do a stack.